lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2537,11 +2559,14 @@ def test_get_anon_ids(self):
         response = self.client.get(url, {})
         self.assertEqual(response['Content-Type'], 'text/csv')
         body = response.content.replace('\r', '')
+
+        print body
+        
         self.assertTrue(body.startswith(
             '"User ID","Anonymized User ID","Course Specific Anonymized User ID"'
-            '\n"3","41","42"\n'
+            '\n"2","41","42"\n'
         ))
-        self.assertTrue(body.endswith('"8","41","42"\n'))
+        self.assertTrue(body.endswith('"7","41","42"\n'))
```

Comment:
So the reason this is like this is because `ModuleStoreTestCase` creates a user by default, and `SharedModuleStoreTestCase` doesn't. That being said, we should just keep a reference to the students and use those IDs instead of hardcoding. I'll fix this.